requirements.txt

```diff
@@ -7,3 +7,5 @@ mpld3==0.5.8
 twitchio==2.4.0
 dotmap==1.3.30
 PyYAML==6.0
+isodate==0.6.1
+requests==2.30.0
```

Comment:
Am I being very silly? Is it normal to have to add this? Feels like a base library?

Reply:
As discussed on discord, use `aiohttp` for all requests, as it's async.